test/models/order_test.rb

```diff
@@ -1,7 +1,12 @@
-require 'test_helper'
+require "test_helper"
 
 class OrderTest < ActiveSupport::TestCase
   test "when Order exists" do
     assert Order
   end
+
+  test "response from bakery if order is done" do
+    orders_for_delivery = Order.completed_orders
```

Comment:
@Rdore88 so yes? it is hitting our production API server in our test environment? This doesn't seem like a good idea? Is the bakery actually producing the orders that your testing?